modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/PersistentPageMemory.java

```diff
@@ -1495,33 +1493,59 @@ private void resetDirtyPages() {
         }
 
         /**
-         * Prepares a page removal for page replacement, if needed.
+         * Tries to replace the page.
          *
-         * @param fullPageId Candidate page full ID.
-         * @param absPtr Absolute pointer of the page to evict.
-         * @return {@code True} if it is ok to replace this page, {@code false} if another page should be selected.
-         * @throws IgniteInternalCheckedException If failed to write page to the underlying store during eviction.
+         * <p>The replacement will be successful if the following conditions are met:</p>
+         * <ul>
+         *     <li>Page has acquired by checkpoint - nothing needs to be done.</li>
```

Comment:
```suggestion
         *     <li>Page has been acquired by checkpoint - nothing needs to be done.</li>
```

Reply:
What do you mean "acquired by checkpoint"? I don't really get it, maybe that's even wrong. Please clarify the statement

Reply:
fix it